pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "bigc"
-version = "0.1.4"
+version = "0.1.5"
```

Comment:
I would remove this, Ryan asked that we bump the version numbers in a separate pr.